### PR TITLE
OpenPAM: new recipe

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

**Comment by mmuman** [comment]
Cool, one thing less on my TODO list!

File: sys-auth/openpam/openpam-20170219.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openpam$secondaryArchSuffix = $portVersion
+	lib:libpam$secondaryArchSuffix = 2.0.0 compat >= 0
+	lib:pam_deny$secondaryArchSuffix = 2.0.0 compat >= 0
+	lib:pam_permit$secondaryArchSuffix = 2.0.0 compat >= 0
+	lib:pam_return$secondaryArchSuffix = 2.0.0 compat >= 0
```
Comment (korli) [inline sys-auth/openpam/openpam-20170219.recipe]: These pam*.so aren't libraries per se. Please remove.

File: sys-auth/openpam/openpam-20170219.recipe [hunk 2]
```diff
@@ -0,0 +1,76 @@
+SUMMARY="An open source PAM library"
+DESCRIPTION="OpenPAM is a Pluggable Authentication Modules implementation."
+HOMEPAGE="http://www.openpam.org/"
+COPYRIGHT="2002-2003 Networks Associates Technology, Inc.
+	2004-2017 Dag-Erling Smørgrav"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://netcologne.dl.sourceforge.net/project/openpam/openpam/Radula/openpam-$portVersion.tar.gz"
```
Comment (korli) [inline sys-auth/openpam/openpam-20170219.recipe]: This isn't a correct URL for sourceforge.

File: sys-auth/openpam/openpam-20170219.recipe [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+CHECKSUM_SHA256="ae8af472bd33c9631a34de56636d30734fc16ce23190d0a991fa15501c090d53"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openpam$secondaryArchSuffix = $portVersion
+	lib:libpam$secondaryArchSuffix = 2.0.0 compat >= 0
```
Comment (korli) [inline sys-auth/openpam/openpam-20170219.recipe]: The compat should probably be >= 2

File: sys-auth/openpam/openpam-20170219.recipe [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+INSTALL()
+{
+	make install
+
+	for f in libpam.la pam_deny.la pam_permit.la pam_return.la; do
+		rm $libDir/$f
+	done
+
+	prepareInstalledDevelLibs libpam pam_deny pam_permit pam_return
```
Comment (korli) [inline sys-auth/openpam/openpam-20170219.recipe]: Please remove pam_deny pam_permit pam_return.

**Pull request merged**